.github/workflows/publish.yml

```diff
@@ -10,10 +10,14 @@ env:
 
 jobs:
   publish:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }} 
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2
+        if: ${{ matrix.os != 'macos-latest' }}
```

Comment:
I take it this is because caching breaks something on the mac runner?

Reply:
Yes, there is some bug with using cache on mac. If you think we should look for workarounds until it's fixed in github actions, I can create a new ticket.

Reply:
No, I think it's fine while publishing is infrequent. If this became more a CD thing we could investigate it